docs/pages/en-us/Error-Handling.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: default
+title: Error Handling
+
+banner:
+    icon: 'assets/img/hard-hat.png'
+    title: Worker Framework
+    subtitle: Cross Platform, Cloud Ready, Microservice Framework
+    links:
+        - title: GitHub
+          url: https://github.com/WorkerFramework/worker-framework
+          
+---
+
+# Error Handling
+
+The purpose of this page is to provide details for the various types of errors and failures that can arise during message processing. A set of rules that every Worker implementation should adhere too is included as well.
+
+## Worker Success
+We start though by describing the expected flow when messages are handled by Worker implementations without any issues at all.
+
+![Success Worker Flow](images/WorkerSuccessFlow.png)
+*Figure 1 - Message successfully processed by Worker*
+
+1. The Worker will receive a message from the messaging input queue, `worker-input-queue` and will start to process it.
+2. The message is successfully processed by the Worker and acknowledged.
+3. A RESULT_SUCCESS response is placed on the messaging output queue, `worker-output-queue` by the worker framework.
```

Comment:
Surely (2) and (3) are mixed up.  The response message should be published before the original message is acknowledged.

Reply:
Yes agreed.